fast-logger/System/Log/FastLogger.hs

```diff
@@ -63,7 +63,7 @@ type FastLogger = LogStr -> IO ()
 -- > {-# LANGUAGE OverloadedStrings #-}
 -- >
 -- > log :: TimedFastLogger -> LogStr -> IO ()
--- > log logger msg = logger (\time -> toLogStr (show time) <> " " <> msg <> "\n")
+-- > log logger msg = logger (\time -> toLogStr $ show time <> " " <> msg <> "\n")
```

Comment:
I think this is not a typo. `(show time) ` is String. `toLogStr` converts it to LogStr. `(<>)` concatenates LogStrs.

Reply:
Oh my bad, I was using a ByteString msg. Sorry for the noise.